docs/css/style.css

```diff
@@ -39,6 +39,11 @@ header {
   padding-top: 20px;
 }
 
+#mocha-logo {
+  width:192;
+  height:192;
```

Comment:
These need a `px` unit affixed, or else they'll be invalid CSS properties:

~~~css
#mocha-logo{
	width: 192px;
	height: 192px;
}

/* (...Or my favourite trick...) */
#mocha-logo{
	width: 1em;
	height: 1em;
	font-size: 192px;
}
~~~

Unitless values are permitted in `width` and `height` attributes largely due to history, IIRC.

Reply:
Agreed. See original issue.
And please not "hairy"'s favourite trick...

Reply:
Just as an aside, the second trick I mentioned is a great way to retain aspect ratio without changing more than one property. E.g., an image with a width 150% greater than its height could could be expressed using `width: 1.5em; height: 1em`.

This only matters when media queries are being used to scale elements snugly between viewport sizes though, so it's moot here.